kernel/allowlist.c

```diff
@@ -181,7 +181,16 @@ void do_load_allow_list(struct work_struct *work)
 	fp = filp_open(KERNEL_SU_ALLOWLIST, O_RDONLY, 0);
 
 	if (IS_ERR(fp)) {
+#ifdef CONFIG_KSU_DEBUG
+		int errno = PTR_ERR(fp);
+		if (errno == -ENOENT) {
```

Comment:
Do we need to check errno here?

Reply:
allow it only when KERNEL_SU_ALLOWLIST does not exist

Reply:
seems reasonable